### PR TITLE
Add support for bonnell-xml package to p10ebmc_config

### PR DESCRIPTION
**Pull request by mabaiocchi**
CANNOT MERGE UNTIL PATCH FILES ARE REMOVED

This commit adds files to pull files from the bonnell-xml repo and build them as part of the p10ebmc_config.  Some existing files were also updated to support this.

### CODE REVIEW

**Review by mabaiocchi — COMMENTED** [review]
CAN'T MERGE UNTIL PATCH FILES ARE REMOVED

pdata patch has been merged in that repo. just needs to be picked up by op-build.

hostboot changes are still under the review process

**Review by debmc — APPROVED** [review]

**Comment by mabaiocchi** [comment]
The pdata and hostboot patch files have been removed.  I verified that this will build the bonnell files with the 1-line command inserted after './op-build p10ebmc_defconfig' has run.  This is safe to be picked up.  Please (re-)review @debmc @fspbld @dcrowell77 @bofferdn .

**Review by debmc — APPROVED** [review]

File: openpower/package/bonnell-xml/bonnell-xml.mk [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+        $(foreach spd_arg, $(SPD_FILES_TOKENED), echo SPD_ARG_ITEMS $(spd_arg);)
+
+        # If we have SPD images the PSPD.bin will be built containing the data, otherwise we get PSPD.bin with all FF's
+        if [ -e "$(BONNELL_XML_MRW_HB_TOOLS)/buildSPDImages.pl" ] && [ -n "$(SPD_IMAGE_FILES)" ]; then \
+            chmod +x $(BONNELL_XML_MRW_HB_TOOLS)/buildSPDImages.pl; \
+            echo "*** RUNNING buildSPDImages.pl ***";\
+            $(BONNELL_XML_MRW_HB_TOOLS)/buildSPDImages.pl --spdOutBin $(BONNELL_XML_MRW_SCRATCH)/$(PSPD_BIN) $(SPD_FILES_TOKENED); \
+        fi
+
+        # Create the MEMD binary
```
Comment (dcrowell77) [inline openpower/package/bonnell-xml/bonnell-xml.mk:176]: Just a note that we don't have MEMD in P10 so this should probably disappear from all of our mk files...  (no need to make a change here though)

**Review by dcrowell77 — APPROVED** [review]

**Comment by mabaiocchi** [comment]
retest this please

File: openpower/package/bonnell-xml/bonnell-xml.mk (hunk 1 above)
Comment (mabaiocchi) [reply openpower/package/bonnell-xml/bonnell-xml.mk:176]: Thank you for this note.  I will remove next week in another PR

**Pull request merged**